.circleci/config.yml

```diff
@@ -41,7 +41,7 @@ step-library:
         name: Prepare
         command: |
           echo "foo" > ~/.mapbox
-          brew install carthage
+          brew update && brew install carthage
```

Comment:
`brew update` just updates the local cache of formulae, and `brew install` would do nothing if (an older version of) Carthage is already installed. This needs to run `brew upgrade`, as in 658e7f8be04c7210af9ddea5919728cb2c30fe0d in #2404.

Reply:
Agreed, thanks.